Composer/packages/client/src/pages/publish/Notifications.tsx

```diff
@@ -71,7 +71,10 @@ export const getPublishedNotificationCardProps = (item: BotStatus): CardProps =>
   };
 };
 
-export const getSkillPublishedNotificationCardProps = (item: BotStatus, url?: string): CardProps => {
+export const getSkillPublishedNotificationCardProps = (
+  item: { status: number; [key: string]: any },
```

Comment:
Record\<string, any>. Can we avoid any here?

Reply:
It just should check status type, so i set any to other properties